packages/components/src/theme/ComposerTheme/Components/Link.ts

```diff
@@ -0,0 +1,36 @@
+import { ComponentStyleConfig } from '@chakra-ui/theme';
+
+export const Link: ComponentStyleConfig = {
+  baseStyle: {
+    color: 'text-link-primary-default',
+    _hover: {
+      textDecoration: 'none',
+      color: 'text-link-primary-hover',
+    },
+    _pressed: {
+      color: 'text-link-primary-pressed',
+    },
+    _visited: {
+      color: 'text-link-primary-pressed',
+    },
+    _focus: {
+      boxShadow: `0 0 0 3px var(--chakra-colors-primary-focus)`,
+      borderRadius: '3px',
+    },
+    _focusVisible: {
+      boxShadow: `0 0 0 3px var(--chakra-colors-primary-focus)`,
+      borderRadius: '3px',
```

Comment:
I believe `borderRadius` should be `4px` instead of `3px`, already double checked with design.